integration

```diff
@@ -326,6 +326,7 @@
   "garbled1/homeassistant_ecowitt",
   "gazoodle/gecko-home-assistant",
   "gcobb321/icloud3",
+  "gcobb321/icloud3_v3",
```

Comment:
Why?
I see no rationale around why this is its own repository instead of just a new release in the existing one.
Its even the same domain name.

Reply:
There are almost 7000 people that have downloaded iCloud3 v2. This is a complete rewrite and, since I am just 1 guy with other things going on, I could not handle questions or issues from a large number of people installing it at the same time. Although there are over 200 people using v3 without problems, creating a separate HACS entry for v3 provides an easy path to reinstall v2 if someone runs into an issue. At some point, when a larger number of people have installed v3, I will upgrade the v2 entry to v3. 

A new v3 entry will help me reach that point and provide an easy installation of v3 using the HACS base repository list instead of having people add a custom repository or downloading a zip file and installing that.

Reply:
That is what I suspected.
Pre-releases would give you the same thing.
I'm going to decline thins inclusion, it can still be used as custom repository if you really want to follow this route.